gateway-notification-service/src/test/java/md/maib/gatewaynotificationservice/config/RoutesConfigTest.java

```diff
@@ -0,0 +1,27 @@
+package md.maib.gatewaynotificationservice.config;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.web.servlet.function.RouterFunction;
+import org.springframework.web.servlet.function.ServerResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+public class RoutesConfigTest {
+
+    @Autowired
+    private ApplicationContext applicationContext;
+
+    @Test
+    public void testNotificationRouteServiceBeanCreated() {
+        // Retrieve the bean from the application context
+        RouterFunction<ServerResponse> bean = applicationContext.getBean("notificationRouteService", RouterFunction.class);
+
+        // Assert that the bean is not null
+        assertThat(bean).isNotNull();
```

Comment:
It's a just null check, need to test if your request pass getaway and return some valid result